.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+indent_style = tab
```

Comment:
Reformatting would be more helpful (not sure which IDE you use).

> I thought you wanted tabs? They were used everywhere.

Not in the old code, just in your added files.